db/fixtures/miq_product_features.yml

```diff
@@ -5123,6 +5123,54 @@
           :description: Edit Job Templates Tags
           :feature_type: control
           :identifier: configuration_script_tag
+
+# Embedded Automation Manager
+- :name: Embedded Automation Manager
+  :description: Everything under Ansible
```

Comment:
@Fryguy this has to match the menus and the menu here says "Ansible".  If we want to change it here to something else, we need to clear that with PM, as the menu title will need to change.  So according to the current mockups, this should be name: Ansible I think.